packages/grant-explorer/src/features/round/ViewCartPage/SummaryContainer.tsx

```diff
@@ -0,0 +1,392 @@
+import { ChainId, PassportState } from "common";
+import { useCartStorage } from "../../../store";
+import React, { useEffect, useMemo, useState } from "react";
+import { BigNumber, ethers } from "ethers";
+import { Summary } from "./Summary";
+import ErrorModal from "../../common/ErrorModal";
+import ChainConfirmationModal from "../../common/ConfirmationModal";
+import { ChainConfirmationModalBody } from "./ChainConfirmationModalBody";
+import { CartProject, ProgressStatus } from "../../api/types";
+import { useQFDonation } from "../../../context/QFDonationContext";
+import { modalDelayMs } from "../../../constants";
+import { useNavigate } from "react-router-dom";
+import { useAccount, useNetwork, useSigner, useSwitchNetwork } from "wagmi";
+import { Logger } from "ethers/lib.esm/utils";
+import { datadogLogs } from "@datadog/browser-logs";
+import { Button } from "common/src/styles";
+import { InformationCircleIcon } from "@heroicons/react/24/solid";
+import { usePassport } from "../../api/passport";
+import useSWR from "swr";
+import _, { round } from "lodash";
+import { getRoundById } from "../../api/round";
+import MRCProgressModal from "../../common/MRCProgressModal";
+import { MRCProgressModalBody } from "./MRCProgressModalBody";
+import { JsonRpcSigner } from "@ethersproject/providers";
+
+export function SummaryContainer() {
+  const { chain } = useNetwork();
+  const { projects } = useCartStorage();
+  const payoutTokens = useCartStorage((state) => state.chainToPayoutToken);
```

Comment:
Not sure how useCartStorage storage works, but is it possible to condense the two lines above in 

```ts
const { projects, payoutTokens } = useCartStorage()
```

?

Reply:
Thou art correct. Fixed.